server/src/db/password_reset_request.rs

```diff
@@ -50,8 +50,8 @@ impl Crud<PasswordResetRequestForm> for PasswordResetRequest {
 impl PasswordResetRequest {
   pub fn create_token(conn: &PgConnection, from_user_id: i32, token: &str) -> Result<Self, Error> {
     let mut hasher = Sha256::new();
-    hasher.input(token);
-    let token_hash: String = PasswordResetRequest::bytes_to_hex(hasher.result().to_vec());
+    hasher.update(token);
+    let token_hash: String = PasswordResetRequest::bytes_to_hex(hasher.finalize().to_vec());
```

Comment:
Haven't tested this, but hopefully its just from an update to Sha256.

Reply:
yup, just a couple method renames from 0.8 -> 0.9